browser/ui/BUILD.gn

```diff
@@ -103,6 +104,18 @@ source_set("ui") {
     "webui/skus_internals_ui.h",
   ]
 
+  if (enable_brave_screenshots) {
+    sources += [
```

Comment:
this should have it's on BUILD.gn and these sources definitely do not belong in `browser/ui/BUILD.gn` because they aren't even in `browser/ui`

Reply:
not sure yet if they should be in `browser/ui` instead, but either way there should be a BUILD.gn for the directory

Reply:
also these sources exist in multiple places so something is definitely weird here if you're not getting any errors

Reply:
oh, there are no errors because they are `.h` files, but you should not be adding files like this

Reply:
I took some time to review this portion of the PR and make (what I hope are) improvements.